inference-engine/src/mkldnn_plugin/mkldnn_node.h

```diff
@@ -336,6 +336,7 @@ class MKLDNNNode {
             selectedPrimitiveDescriptorIndex = -1;
         else
             selectedPrimitiveDescriptorIndex = index;
+        inplace = InPlaceType::Unknown;
```

Comment:
In general this is not clear why we need to change inplace state inside selectPrimitiveDescriptorByIndex. Maybe it is beteer to leave brief comment here.

Reply:
Added comment and rebased on the latest master